search_params.py

```diff
@@ -165,15 +165,15 @@ def prepare_retrain_config(best_config, best_log_dir, merge_train_val):
         optimal_idx = log_metric.argmax() if best_config.mode == "max" else log_metric.argmin()
         best_config.epochs = optimal_idx.item() + 1  # plus 1 for epochs
     else:
-        best_config.merge_train_val = False
+        best_config.retrain = False
```

Comment:
Is `best_config.merge_train_val` used in main.py? ([link](https://github.com/ASUS-AICS/LibMultiLabel/blob/f325f4877c91751c2075307eb2b7bc486892e5e2/main.py#L48))